docs/sdd/requirements/network-filtering/stories/US-006.md

```
@@ -0,0 +1,45 @@
+  - npm: *.npmjs.org:443, *.npmjs.com:443
+  - GitHub: *.github.com:443, *.githubusercontent.com:443
+  - PyPI: pypi.org:443, *.pythonhosted.org:443
+  - Docker Hub: *.docker.io:443, *.docker.com:443
+
+### AC-003: 環境ごとのカスタマイズ
+
+- 自動適用されたルールは、通常のルールと同様に編集・削除・無効化できる
+- 追加のルールも自由に設定可能
+- フィルタリング自体を無効にすることも可能
+
+### AC-004: 既存環境への影響なし
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)